gms/client/src/main/java/com/linkedin/common/client/MLModelsClient.java

```diff
@@ -0,0 +1,26 @@
+package com.linkedin.common.client;
+
+import com.linkedin.common.urn.MLModelUrn;
+import com.linkedin.ml.MLModelKey;
+import com.linkedin.restli.client.Client;
+import javax.annotation.Nonnull;
+
+public abstract class MLModelsClient extends BaseClient {
```

Comment:
I was copying the pattern from the other clients. This is the same as how Datasets & DatasetsClient are set up. I'm more than happy to have it implemented that way, just want to make sure that it's okay to break pattern there.

Reply:
So `[DatasetsClient](https://github.com/linkedin/datahub/blob/master/gms/client/src/main/java/com/linkedin/common/client/DatasetsClient.java)` seems to be extended by other clients like `[Deprecations](https://github.com/linkedin/datahub/blob/master/gms/client/src/main/java/com/linkedin/dataset/client/Deprecations.java)`, `[Schemas](https://github.com/linkedin/datahub/blob/master/gms/client/src/main/java/com/linkedin/dataset/client/Schemas.java)`, etc hence it makes sense to create a separate one for reusability. But `[CorpUsersClient](https://github.com/linkedin/datahub/blob/master/gms/client/src/main/java/com/linkedin/common/client/CorpUsersClient.java)` doesn't seem to be used anywhere - will be working on a PR to get rid of it.

Appreciate your patience here!

Reply:
No worries! Since I'm not using it for other client resources I'll go ahead and just add the methods to MLModels. Can always be pulled out later if ML models need things like Deprecations/Schemas.